README.md

```diff
@@ -7,6 +7,19 @@ available on Uppmax, and the individual READMEs may also be incomplete in terms
 of what was actually done to install the modules.  We are publicising these in
 the hopes that they can be helpful.
 
+Example workflow of a basic installation
+-------
+1. Clone the install methods git repo (`git clone https://github.com/UPPMAX/install-methods.git`)
+1. Add the repo to your `$PATH` and source the `uppmax_functions.sh` file to get access to the functions.
+1. Run `run_makeroom` with at least `-t` and `-v`, to generate a `.sh` (`makeroom_toolname_version.sh`) file that will create the directory structure needed in `/sw`
+1. Run the `.sh` file created in the directory you are standing to create the directory structure (`/sw/category/toolname/` and `/sw/mf/common/category`) and template files.
+1. Put the source code for the program in `/sw/category/toolname/version/src`
+1. Compile and/or install the tool in `/sw/category/toolname/version/cluster/bin` etc.
+1. Edit the readme file, explaining how you did the installation, in `/sw/category/toolname/toolname-version_install-README.md`
```

Comment:
18.1 run fixup on the cluster directory if needed. fixup sets the rigth GID and sets the correct permissions. This is run by makeroom, but it may be modified during tool installation.

Reply:
Nice, added it and will push it shortly.